### PR TITLE
Web f2 status table v1, connects #2

### PR DESCRIPTION
**Pull request by RamitaTr**
## :rocket: What does this PR implement/fix?  
- Created and styled parking status table, status summary, and legend components  

## :book: Where should the reviewers start?  
- Prop-types package is added, need to run `npm install`   
- Check the following file:  
    - ParkingStatus.js  
    - StatusTable.js  
    - StallStatus.js
    - StatusSummary.js
    - StatusLegend.js    

## :movie_camera: Screencast/Demo  
image

## :bulb: Additional context you want to provide?  
- This is version 1 of UI, the improvement will be developed in the future version  
- For QA:  
    - git clone and checkout web-f2-status-table-v1  
    - npm run storybook

### CODE REVIEW

File: src/StatusSummary.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  background: goldenrod;
+`
+
+const CellRed = styled(CellGreen)`
+  background: indianred;
+`
+
+const StatusSummary = ({data}) => {
+  let vacant = 0
+  let ocupied = 0
```
Comment (tomliangg) [inline src/StatusSummary.js]: spelling: `ocupied` -> `occupied`

File: src/StatusSummary.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+            <CellRed>ocupied</CellRed><CellRed>{ocupied}</CellRed>
+          </tr>
+        </tbody>
+      </Table>
+    </div>
+  )
+}
+
+StatusSummary.propTypes = {
+  data: PropTypes.array
```
Comment (tomliangg) [inline src/StatusSummary.js]: You can chain the proptype with `isRequired` to make sure a warning is shown if the prop isn't provided
-> `data: PropTypes.array.isRequired`

File: src/StatusTable.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+            )
+          })
+        }
+      </tbody>
+    </Table>
+  )
+}
+
+StatusTable.propTypes = {
+  data: PropTypes.array
```
Comment (tomliangg) [inline src/StatusTable.js]: `data: PropTypes.array.isRequired`

File: src/StatusSummary.js (hunk 1 above)
Comment (tomliangg) [reply src/StatusSummary.js]: Notice the spelling mistakes in a few places. Can do a global search and replace all `ocupied` to `occupied`

**Comment by tomliangg** [comment]
## :bug: Steps to reproduce  
- take a look at the Parking status component  
- notice the alignment is not consistent (i.e. the left column, the 

## :camera: Screenshots of errors  
image

## :bulb: Additional info you want to provide  
- info1  
- info2

**Comment by MinhNguyen911** [comment]
## :bug: Steps to reproduce  
- Look at the parking map page UI

## :camera: Screenshots of errors  
image


## :bulb: Additional info you want to provide  
- Unsymmetrical parking spot's ID numbers (first column v.s second column)

**Comment by RamitaTr** [comment]
@MinhNguyen911 The error should be fixed, could you run the test to see if the error is reproducible?  
- git fetch and checkout web-f2-status-table-v1  
- git pull  
- npm run storybook

**Review by tomliangg — APPROVED** [review]

**Comment by MinhNguyen911** [comment]
## :bug: Steps to reproduce  
- The word 'Summmary' : wrong spelling

## :camera: Screenshots of errors  
image

**Comment by RamitaTr** [comment]
@MinhNguyen911 Fixed the typo, could you do git pull and check if it is reproducible?

**Comment by MinhNguyen911** [comment]
> @MinhNguyen911 Fixed the typo, could you do git pull and check if it is reproducible?

It is fixed now.

**Pull request merged**